adapters/risc0/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = { workspace = true }
 homepage = { workspace = true }
 license = { workspace = true }
 repository = { workspace = true }
-rust-version = { workspace = true }
```

Comment:
nit: consider not adding a blank line here so the header is concise

Reply:
Fixed